cypress.json

```diff
@@ -6,7 +6,7 @@
     "video": false,
     "env": {
         "dhis2DataTestPrefix": "dhis2-scheduler",
-        "dhis2ApiVersion": "41",
+        "dhis2ApiVersion": "42",
```

Comment:
This needs a `BREAKING CHANGE`-commit right? Or would you squash-merge and add that in the message? https://github.com/dhis2/scheduler-app/pull/605/commits/6756afaa2c82d1acc19b2ab55bd9b7ef71416e0d doesn't seem to contain this.